apiary.apib

```diff
@@ -646,7 +646,9 @@ Stores input as an IPFS block
 
     In the example provided, the 'test' file has the following contents:
 
+        ```
         hello world
+        ```
```

Comment:
You're killing me Friedel. 

Either we go with backticks at this level, or I change everything.